android/Staccato_AN/app/src/main/java/com/on/staccato/presentation/common/categoryselection/CategorySelectionFragment.kt

```diff
@@ -81,5 +84,6 @@ class CategorySelectionFragment : BottomSheetDialogFragment() {
 
     companion object {
         const val TAG = "CategorySelectionModalBottomSheet"
+        const val MESSAGE_IMPLEMENT_VIEWMODEL_EXCEPTION = "Activity must implement ProvidesCategorySelectorViewModel"
```

Comment:
앗 혹시 메시지가 아래처럼 되어야 하나요?

```suggestion
        const val MESSAGE_IMPLEMENT_VIEWMODEL_EXCEPTION = "Activity must implement CategorySelectionViewModelProvider"
```